api/v2/dogu_types.go

```diff
@@ -70,6 +70,41 @@ type DoguSpec struct {
 	UpgradeConfig UpgradeConfig `json:"upgradeConfig,omitempty"`
 	// AdditionalIngressAnnotations provides additional annotations that get included into the dogu's ingress rules.
 	AdditionalIngressAnnotations IngressAnnotations `json:"additionalIngressAnnotations,omitempty"`
+	// Data provides the possibility to mount additional data into the dogu.
+	// +optional
+	Data []DataMount `json:"data,omitempty" patchStrategy:"replace"` // no unique identifier, so we can't use merge
```

Comment:
Can we  use a better name for it? `data` is too unspecific
```suggestion
	Data []DataMount `json:"additionalMounts,omitempty" patchStrategy:"replace"` // no unique identifier, so we can't use merge
```